.github/workflows/main.yml

```diff
@@ -10,8 +10,12 @@ jobs:
   tests:
     if: github.event.action != 'closed' || github.event.pull_request.merged
 
+    name: Test Python ${{ matrix.python_version }}
     runs-on: ubuntu-latest
-
+    strategy:
+      max-parallel: 4
+      matrix:
+        python_version: ['3.5', '3.6', '3.7', '3.8']
```

Comment:
3.5 isn't worth it; it goes away in less than 6 months.

Reply:
Coolbeans!